Grid/grid.css

```diff
@@ -0,0 +1,23 @@
+@import url("/styles/reset.css");
+@import url("/styles/utility.css");
+
+.container {
+    margin: 2rem;
+}
+
+.grid-container-two-col {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+}
+
+.grid-container-three-col {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+}
+
+.box {
+    border: 1px solid black;
```

Comment:
You can declare the color variables in your root

Reply:
thank you for pointing out this, I missed adding color variable over here

Reply:
updated, thanks